products/big-ip.md

```diff
@@ -0,0 +1,44 @@
+---
+title: BIG-IP 
+category: os
+permalink: /big-ip
+versionCommand: show /sys version
+releaseDateColumn: true
+eolColumn: End of Technical Support
+eoasColumn: End of Software Development
+
+releases:
+
+-   releaseCycle: "17.1.x"
+    releaseDate: 2023-03-14
+    eoas: 2027-03-31
+    eol: 2027-03-31
+    latest: "17.1.1"  
+    lts: true
+
+-   releaseCycle: "16.1.x"
+    releaseDate: 2021-07-07
+    eoas: 2025-07-31
+    eol: 2025-07-31
+    latest: "16.1.5"
+    lts: true
+
+-   releaseCycle: "15.1.x"
+    releaseDate: 2019-12-11
+    eoas: 2025-12-31
+    eol: 2025-12-31
+    latest: "15.1.10"
+    lts: true
+
+---
+
+> [BIG-IP](https://www.f5.com/products/big-ip) is a family of hardware and software products from F5 Networks, designed for load balancing, traffic management, and application security.
+
+
+**Summary of the release policy**:  
```

Comment:
```suggestion
```

No need for the title : the first paragraph is always the release policy on endoflife.date.